cli/command/formatter/container.go

```diff
@@ -115,7 +116,24 @@ func (c *containerContext) Image() string {
 		if trunc := stringid.TruncateID(c.c.ImageID); trunc == stringid.TruncateID(c.c.Image) {
 			return trunc
 		}
+		// truncate digest if no-trunc option was selected
+		ref, err := reference.ParseNormalizedNamed(c.c.Image)
+		if err == nil {
+			if nt, ok := ref.(reference.NamedTagged); ok {
+				// case for when a tag is provided
+				if namedTagged, err := reference.WithTag(reference.TrimNamed(nt), nt.Tag()); err == nil {
+					return reference.FamiliarString(namedTagged)
+				}
+			} else {
+				// case for when a tag is not provided
+				if n, ok := ref.(reference.Named); ok {
```

Comment:
Actually, this check is not needed. `ParseNormalizedNamed` returns a `reference.Named`.